python/Ganga/Core/GangaRepository/SessionLock.py

```diff
@@ -959,3 +841,4 @@ def session_to_info(self, session):
         except Exception, err:
             logger.debug( "Session Info Exception: %s" % err)
             return session
+
```

Comment:
Apologies this is the warning number from a common piece of software for checking the code, I forget which. I was having issues with a certain editor adding whitelines to files in the past. obv not the case here it's an extra line but this wasn't clear from the github interface.